draft-ietf-quic-transport.md

```diff
@@ -3797,17 +3797,23 @@ later time in the connection.
 The QUIC packet size includes the QUIC header and protected payload, but not the
 UDP or IP header.
 
+QUIC defines the minimum QUIC packet size as at least 1280 bytes.
```

Comment:
```suggestion
QUIC depends upon a minimum packet size as at least 1280 bytes.
```

Reply:
Fine, except shouldn't this be: /as at/of at/